homeassistant/components/spotify/media_player.py

```diff
@@ -247,33 +249,14 @@ def __init__(
         self._currently_playing: dict | None = {}
         self._devices: list[dict] | None = []
         self._playlist: dict | None = None
-        self._spotify: Spotify = None
-
-        self.player_available = False
-
-    @property
-    def name(self) -> str:
-        """Return the name."""
-        return self._name
-
-    @property
-    def icon(self) -> str:
-        """Return the icon."""
-        return ICON
 
-    @property
-    def available(self) -> bool:
-        """Return True if entity is available."""
-        return self.player_available
-
-    @property
-    def unique_id(self) -> str:
-        """Return the unique ID."""
-        return self._id
+        self._attr_name = self._name
+        self._attr_unique_id = user_id
 
     @property
     def device_info(self) -> DeviceInfo:
         """Return device information about this entity."""
+        model = "Unknown"
```

Comment:
We normally don't set the model item in the device info at all if it's unknown.

Reply:
Would setting it to `None` be accepted?

Reply:
Never mind, my mind isn't straight, will adjust.

Reply:
Yes, that's the same as the default:
https://github.com/home-assistant/core/blob/5afd16ef5d45150e160f488e2e60f9de7ab521b6/homeassistant/helpers/device_registry.py#L60

Reply:
But it wasn't empty, it is always set. It was just weird logic. Made it simpler and more clear now.